packages/core/src/App/Containers/CashierModal/PaymentAgent/payment-agent-receipt.jsx

```diff
@@ -24,64 +26,72 @@ class PaymentAgentReceipt extends React.Component {
     render() {
         const payment_agent = this.props.receipt.payment_agent_name || this.props.receipt.payment_agent_id;
         return (
-            <React.Fragment>
-                <div className='payment-agent__receipt'>
-                    <h2 className='cashier__header'>
-                        <Localize
-                            i18n_default_text='Your funds have been transferred to {{payment_agent}}.'
-                            values={{ payment_agent }}
-                            options={{ interpolation: { escapeValue: false } }}
-                        />
-                    </h2>
-                    <div className='cashier__transferred-amount cashier__text--bold'>
-                        <span
-                            className={classNames('symbols', `symbols--${this.props.currency.toLowerCase()}`)}
-                        />
-                        {this.props.receipt.amount_transferred}
-                    </div>
-                    <div className='cashier__transferred-details-wrapper'>
-                        <Icon icon='IconAccountsCurrency' type={this.props.currency.toLowerCase()} />
-                        <span className='cashier__transferred-details'>
-                            <span className='cashier__text--bold'>{this.props.currency}</span>&nbsp;({this.props.loginid})
-                        </span>
-                        <Icon className='cashier__transferred-icon' icon='IconBack' />
-                        <Icon icon='IconPaymentAgent' />
-                        <span className='cashier__transferred-details'>
-                            {this.props.receipt.payment_agent_name && <span className='cashier__text--bold'>{this.props.receipt.payment_agent_name}&nbsp;</span>}({this.props.receipt.payment_agent_id})
-                        </span>
+            <div className='cashier__wrapper--align-left'>
+                <ThemedScrollbars
+                    autoHide
+                    autoHeightMax={550}
+                    renderTrackHorizontal={props => <div {...props} className='track-horizontal' style={{ display: 'none' }} />}
+                    renderThumbHorizontal={props => <div {...props} className='thumb-horizontal' style={{ display: 'none' }} />}
```

Comment:
Are these props necessary still with `ThemedScrollbars`? `ThemedScrollbars` in Components seems to have this built-in.